calendar-bundle/src/Resources/contao/dca/tl_calendar_events.php

```diff
@@ -250,7 +250,6 @@
 		'endDate' => array
 		(
 			'label'                   => &$GLOBALS['TL_LANG']['tl_calendar_events']['endDate'],
-			'default'                 => null,
 			'exclude'                 => true,
 			'inputType'               => 'text',
 			'eval'                    => array('rgxp'=>'date', 'doNotCopy'=>true, 'datepicker'=>true, 'tl_class'=>'w50 wizard'),
```

Comment:
should probably add a `nullIfEmpty` flag?

Reply:
Doesn’t seem to be necessary, if I don’t specify an end date in the backend `NULL` gets stored in the database already.